### DISCUSSION TITLE
Plugin not found in Marketplace

### DISCUSSION THREAD

**Discussion by Luktronic** [doki-theme/doki-theme-icons-jetbrains#13] [Q&A] [answered]
Hey there! 👋 

I have IntelliJ build 223 installed and cannot seem to find the Doki Theme Icon plugin through the IDEA Marketplace.

![image](https://user-images.githubusercontent.com/59698507/205741426-315bb385-be3d-4f49-b49e-4b43705a875e.png)

When searching, only the regular Doki Theme Plugin is showing up.
Also, when I tried to install the Plugin manually, it threw some exceptions, but that's probably a topic for another discussion.

So what could be the reason for the plugin not showing up? Is this intentional? Am I doing something wrong?
Thanks in advance 😄

**Comment by Unthrottled** [comment, marked as answer]
> So what could be the reason for the plugin not showing up? Is this intentional? Am I doing something wrong?

You are on the bleeding edge friend!

I am still in the process of vetting things out. I've cut a release, but I'm using it in my daily work to make sure things are okay.

You'll need to install the Doki Theme 88.2-1.6.0+ to use the icons, free link to download and manual install https://github.com/doki-theme/doki-theme-jetbrains/releases/tag/v88.3-1.7.1

I still haven't got things up to snuff with the Doki Theme, but once I do. I'll publish the icons then the version of Doki Theme that works with the icons.

So to answer your question, you arrived early to the party 😄 🎊

**Reply by Luktronic** [reply]
Alright then, thanks for the answer!

Maybe you could put a **Not yet released** on top of the README then? So other people don't get confused 🙂

**Reply by Unthrottled** [reply]
See that would have required effort 😄 

I made the update, thanks!

**Reply by Unthrottled** [reply]
As an FYI, I had to jump through some hoops, but is available in the marketplace now. 🎉

**Reply by Luktronic** [reply]
🥳
